Update branding to 9.0.108

vseanreesermsft opened the pull request.

Commit:
Fix minimum width of Author column in tabular output

vseanreesermsft committed:
Update branding to 9.0.108

marcpopMSFT committed:
Update source build to latest 17.8 version

marcpopMSFT commented:
@dotnet/source-build The source build leg was getting flagged for a known vulnerability in msbuild 17.8.3. Do you know why source build is using a 17.8 msbuild when 9.0 shipped with 17.12? I updated it to 17.8.29 as the latest 17.8 version but wanted to double check that it should even be a 17.8 version.

v-wuzhai commented:
I noticed this looks like the same issue being discussed in issue #48964, and it seems like @NikolaMilosavljevic is already following up on it — maybe we can move the discussion over there instead?

NikolaMilosavljevic commented:
> I noticed this looks like the same issue being discussed in issue #48964, and it seems like @NikolaMilosavljevic is already following up on it — maybe we can move the discussion over there instead?

The other, linked issue was about Arcade's use of MSBuild packages at 17.8 level - I have updated the Arcade dependencies with https://github.com/dotnet/arcade/pull/15843 - I'll check why this is still being flagged.

NikolaMilosavljevic commented:
> > I noticed this looks like the same issue being discussed in issue #48964, and it seems like @NikolaMilosavljevic is already following up on it — maybe we can move the discussion over there instead?
> 
> The other, linked issue was about Arcade's use of MSBuild packages at 17.8 level - I have updated the Arcade dependencies with [dotnet/arcade#15843](https://github.com/dotnet/arcade/pull/15843) - I'll check why this is still being flagged.

Last arcade flow was on 5/13 - https://github.com/dotnet/sdk/commit/5c414b76dd665d0b040cce4614a48a5f7120b1e9 My fix for MSBuild dependencies in arcade was merged on 5/16. Perhaps the subscription is manual - @marcpopMSFT @mmitche

mmitche commented:
The subscriptions are set to manual trigger for Arcade. We usually trigger them on a monthly update on Patch Tuesday. We can trigger this for 9.0 now though, no problem.

mmitche commented:
https://github.com/dotnet/sdk/pull/48964 from a few weeks ago

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/nuget/nuget.client build 6.12.4.9

Microsoft.Build.NuGetSdkResolver , NuGet.Build.Tasks , NuGet.Build.Tasks.Console , NuGet.Build.Tasks.Pack , NuGet.CommandLine.XPlat , NuGet.Commands , NuGet.Common , NuGet.Configuration , NuGet.Credentials , NuGet.DependencyResolver.Core , NuGet.Frameworks , NuGet.LibraryModel , NuGet.Localization , NuGet.Packaging , NuGet.ProjectModel , NuGet.Protocol , NuGet.Versioning
 From Version 6.12.4-rc.1 -> To Version 6.12.4-rc.9

marcpopMSFT committed:
Update arcade version to try to unblock source build
